lib/fileio/chaser/chaser.cpp

```diff
@@ -0,0 +1,42 @@
+//
```

Comment:
This file was newly added when it should have been moved from plugin/pxr/maya/lib/usdMaya.

The file is still present in the old location and should be removed:
https://github.com/Autodesk/maya-usd/blob/Autodesk/ppt/refactoring_sandbox_to_dev_00/plugin/pxr/maya/lib/usdMaya/chaser.cpp